lib/MusicBrainz/Server/Data/Edit.pm

```diff
@@ -511,16 +511,16 @@ sub create {
 
     $edit->insert;
 
-    my $now = DateTime->now;
     my $duration = DateTime::Duration->new( days => $conditions->{duration} );
+    my $interval = DateTime::Format::Pg->format_interval($duration);
 
     my $row = {
         editor => $edit->editor_id,
         data => JSON::Any->new( utf8 => 1 )->objToJson($edit->to_hash),
         status => $edit->status,
         type => $edit->edit_type,
-        open_time => $now,
-        expire_time => $now + $duration,
+        open_time => \"now()",
+        expire_time => \"now() + interval '$interval'",
```

Comment:
Oh, and the `\` in front of the string creates a scalar reference, which tells `insert_row` to insert the string directly into the query, not to use `?` and parameter substitution, which obviously wouldn't work there.

Something new I learned today. :)

See https://github.com/mwiencek/musicbrainz-server/blob/mbs-1880/lib/Sql.pm#L119